Makefile

```diff
@@ -25,3 +24,6 @@ js/test: js/vimlparser.js
 
 py/test: py/vimlparser.py
 	test/run_command.sh python py/vimlparser.py
+
+clean:
```

Comment:
What's the difference between clean_compiled and clean?

Maybe we can merge it. (remove clean_compiled)